### ISSUE TITLE
native extension error at installing

### ISSUE THREAD

**Issue by miniway** [treasure-data/embulk-output-td#43]
We got an error at installing

```
# embulk gem install td
2016-05-31 17:33:30.083 +0900: Embulk v0.8.9
WARNING: SSLSocket#session= is not supported

Building native extensions.  This could take a while...
ERROR:  Error installing td:
    ERROR: Failed to build gem native extension.

   java -cp /root/.embulk/bin/embulk org.jruby.Main -r ./siteconf20160531-13525-1jwldbh.rb extconf.rb
NotImplementedError: C extensions are not supported
   <top> at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/mkmf.rb:1
 require at org/jruby/RubyKernel.java:937
  (root) at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
   <top> at extconf.rb:1

extconf failed, exit code 1
```

Is it something to do with https://github.com/embulk/embulk/pull/429 ?

**Comment by muga** [comment]
@miniway You might hit same issue as https://github.com/jruby/jruby/issues/3635. If so, we should upgrade JRuby version that Embulk has used ASAP. Thank you for pinging me.

**Comment by muga** [comment]
@miniway sorry for the delay. https://github.com/embulk/embulk/pull/429 was already merged and released as v0.8.10. The bug might be able to be fixed. Please let me know if you have any problems or concerns.
